app/modules/time/TimeService.js

```diff
@@ -10,7 +10,16 @@ var TimeService = {
     var hour = (d.getHours() >= 10) ? (d.getHours()) : ('0'+d.getHours())
     var min = (d.getMinutes() >= 10) ? (d.getMinutes()) : ('0'+d.getMinutes())
     return year+'/'+month+'/'+day+' '+hour+':'+min
-  }
+  },
+
+  isDate: function (obj) {
+    if (typeof obj == 'undefined' || obj == this.NO_DATE)
+      return false
+    
+    return true
+  },
+    
+  NO_DATE: '3000-01-01T00:00:00.000Z'
```

Comment:
Napotkałem następujący problem:
Jeśli wybrałem datę z kalendarza, to wyświetlała się ona ok. Natomiast jeśli odznaczyłem checkbox związany z datą, to poprzednia data nie została usunięta z bazy danych i wyświetlała się poprzednia data. Dlatego postanowiłem w przypadku odznaczenia checkboxa przyjmować jakąś datę z odległej przyszłości".